lib/internal/util/inspect.js

```diff
@@ -2470,12 +2475,20 @@ function formatPromise(ctx, value, recurseTimes) {
   return output;
 }
 
+function formatExtraProperties(ctx, value, recurseTimes, key, typedArray) {
+  ctx.indentationLvl += 2;
+  const str = formatValue(ctx, value[key], recurseTimes, typedArray);
+  ctx.indentationLvl -= 2;
+
+  // These entries are mainly getters. Should they be formatted like getters?
+  return ctx.stylize(`[${key}]: ${str}`, 'string');
+}
+
 function formatProperty(ctx, value, recurseTimes, key, type, desc,
                         original = value) {
   let name, str;
   let extra = ' ';
-  desc ||= ObjectGetOwnPropertyDescriptor(value, key) ||
-    { value: value[key], enumerable: true };
```

Comment:
FYI @BridgeAR / @aduh95 I think the removal of the fallback to `{ value: value[key], enumerable: true }` has caused a regression, where inspecting previously inspect-able values will throw an exception: https://github.com/nodejs/node/issues/60717